tests/FSharpPlus.Tests/General.fs

```diff
@@ -580,6 +612,14 @@ module Indexable =
 
         ()
 
+    [<Test>]
+    let tryItemReadonly() =
+        let d = ReadOnlyDictionary( Map.ofSeq [1, "one"; 2, "two"] :> IDictionary<_,_> )
```

Comment:
Here you can just do `dict [1, "one"; 2, "two"]` and you'll get the IDictionary

Reply:
Thanks! 👍